tools/codemods/fix_bare_except.py

```
@@ -0,0 +1,98 @@
+
+EXCEPT_PATTERNS = [
+    # "except Exception: pass" - replace with proper exception handling
+    re.compile(r"(\s*)except\s+Exception\s*:\s*pass\s*$", re.MULTILINE),
+    # "except Exception:" followed by indented content - replace with proper exception handling
+    re.compile(r"(\s*)except\s+Exception\s*:\s*$", re.MULTILINE),
+]
+
+
+def ensure_logging_import(text: str) -> str:
+    lines = text.splitlines()
+    for i, line in enumerate(lines[:30]):  # ищем в первой трети для скорости
```

Ruff (0.13.3)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)